DEVELOPMENT.md

```diff
@@ -63,8 +71,9 @@ b) Run the dev server for each language with `yarn start -- --locale en` (Replac
 
 ### Deploy (Staff Use Only)
 
-This is only for preparing the documentation for deployment to GitHub Pages.
+While the section below may still be useful, its worth noting that we have automated this process with github actions, now a build we automatically be deployed anytime a push is made to main, aswell as once every 24hr's
```

Comment:
its -> it's
we automatically -> will automatically
aswell -> as well
not sure about 24hr's, maybe 24hrs?

Reply:
Fixed in latest commit 😄